noetic/distribution.yaml

```diff
@@ -7288,6 +7288,12 @@ repositories:
       url: https://github.com/Jmeyer1292/opw_kinematics.git
       version: master
     status: developed
+  ortools_vendor:
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/ros2-gbp/ortools_vendor-release.git
```

Comment:
I noticed this is in `ros2-gbp`, but this PR is releasing into ROS Noetic which is ROS 1. I see it's using ament_cmake, so I'm wondering if this release into ROS Noetic was an accident.

https://github.com/ros2-gbp/ortools_vendor-release/blob/debian/noetic/focal/ortools_vendor/CMakeLists.txt

Reply:
It was intentional